api/controllers/NetworkController.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
       .tap(network => Promise.map(req.param('communities'), communityId =>
         Membership.hasModeratorRole(req.session.userId, communityId)
         .then(isModerator => {
-          if (isModerator) {
+          if (isModerator || Admin.isSignedIn(req)) {
             return Community.find(communityId)
```

Comment:
this can be done without the community lookup -- just `Community.query().where('id', communityId).update({network_id: network.id})`